gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
```

Comment:
have you used the `wrapper` task to do this? I'm just asking because sometimes that updates also the wrapper scripts and such

Reply:
No I didn't. Will do that 👍

Reply:
It changed the path from `distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip` to `distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-bin.zip` 🤔

Reply:
we actually want `all.zip`